MAUI/TimePicker/getting-started.md

```diff
@@ -0,0 +1,189 @@
+
+            builder.ConfigureSyncfusionCore();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("Segoe-mdl2.ttf", "SegoeMDL2");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set header to the Time Picker
+
+The SfTimePicker control allows you to add the header text by setting the [Text] property in the [PickerHeaderView]. To enable the header view by setting the [Height] property in [PickerHeaderView].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3" %}
+
+<picker:SfTimePicker x:Name="picker">
+    <picker:SfTimePicker.HeaderView>
+        <picker:PickerHeaderView Text="Time Picker" Height="40" />
+    </picker:SfTimePicker.HeaderView>
+</picker:SfTimePicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="4 5" %}
+
+SfTimePicker picker = new SfTimePicker();
+picker.HeaderView = new PickerHeaderView()
+{
+    Text = "Time Picker",
+    Height = 40,
+};
+
+this.Content = picker;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set footer to the Time Picker
+
+In SfTimePicker control, validation buttons (OK and Cancel) can be customized by setting the [OkButtonText] and [CancelButtonText] property in the [PickerFooterView]. It allows you to confirm or cancel the selected time.  The `OkButtonText` can be enabled by using the [ShowOkButton] property in [PickerFooterView].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3" %}
+
+<picker:SfTimePicker x:Name="Picker">
+    <picker:SfTimePicker.FooterView>
+        <picker:PickerFooterView ShowOkButton="True" Height="40" />
+    </picker:SfTimePicker.FooterView>
+</picker:SfTimePicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="4 5" %}
+
+SfTimePicker picker = new SfTimePicker();
+picker.FooterView= new PickerFooterView()
+{  
+    ShowOkButton = true,
+    Height = 40,
+};
+
+this.Content = picker;
+
+{% endhighlight %}  
+{% endtabs %}
+
+## Set height and width to the Time Picker
+
+The SfTimePicker control allows you to the change the height and width by using the [HeightRequest] and [WidthRequest] property in [SfTimePicker].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3 5" %}
+
+<picker:SfTimePicker x:Name="Picker" 
+                    HeightRequest="280" 
+                    WidthRequest="300">
+</picker:SfTimePicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="3 4" %}
+
+SfTimePicker picker = new SfTimePicker()
+{
+    HeightRequest = 280,
+    WidthRequest = 300,
+};
+
+this.Content = picker;
+
+{% endhighlight %}  
+{% endtabs %}
+
+## Set selected time to the Time Picker
+
+The SfTimePicker control allows you to select the time by using the [SelectedTime] property in [SfTimePicker]. The default value of the `SelectedTime` is current time.
```

Comment:
The SfTimePicker control allows you to select the time using the [SelectedTime] property in the [SfTimePicker]. The default value of the `SelectedTime` is the current time.

Reply:
Content modified.